### PR TITLE
Update workflow and workspace templates

### PR DESCRIPTION
**Pull request by savan77**
1) Updated existing templates
2) Added template for object detection evaluation

### CODE REVIEW

**Comment by rushtehrani** [comment]
@savan77 can you resolve the conflict? Note the filesyncer is commented out in master.

**Comment by savan77** [comment]
@rushtehrani can you take a look and merge?

File: workflows/maskrcnn-training/template.yaml [hunk 1]
```diff
@@ -41,25 +41,26 @@ templates:
...
-#        - from: '{{tasks.train-model.outputs.artifacts.sys-metrics}}'
-#          name: metrics
-#        parameters:
-#        - name: status
-#          value: '{{tasks.train-model.status}}'
-#      dependencies:
-#      - train-model
-#      name: notify-in-slack
-#      template: slack-notify-success
+    # Uncomment following lines to get Slack notifications
```
Comment (rushtehrani) [inline workflows/maskrcnn-training/template.yaml:44]: This should be commented with the same indentation as before.

File: workflows/tf-object-detection-training/template.yaml [hunk 2]
```diff
@@ -43,31 +43,29 @@ templates:
...
-#          name: metrics
-#        parameters:
-#        - name: status
-#          value: '{{tasks.train-model.status}}'
-#      dependencies:
-#      - train-model
-#      name: notify-in-slack
-#      template: slack-notify-success
+    # Uncomment following lines to get Slack notifications
+    # - arguments:
```
Comment (rushtehrani) [inline workflows/tf-object-detection-training/template.yaml:47]: Same here, the comment should be in the first position.

File: workflows/maskrcnn-training/template.yaml (hunk 1 above)
Comment (savan77) [reply workflows/maskrcnn-training/template.yaml]: ok, updated.

File: workflows/maskrcnn-training/template.yaml [hunk 3]
```diff
@@ -85,18 +87,12 @@ templates:
     - mountPath: /mnt/output
       name: output
     workingDir: /mnt/src
-  nodeSelector:
-    beta.kubernetes.io/instance-type: '{{workflow.parameters.sys-node-pool}}'
```
Comment (rushtehrani) [inline workflows/maskrcnn-training/template.yaml:89]: We can put this back in, the new API supports these properly now.

Comment (savan77) [reply workflows/maskrcnn-training/template.yaml]: ok, updated templates.

**Pull request merged**